### ISSUE TITLE
Add link to our GitHub organization

### ISSUE THREAD

**Issue by WasabiFan** [ev3dev/ev3dev.github.io#251]
From ev3dev/ev3dev#734.

**Comment by vini2710** [comment]
Maybe there could be a like somewhere in the header?

**Comment by WasabiFan** [comment]
Yeah, that might be where I put ou it. I haven't started on this tes tes yet so I don't know exactly where it will go.

**Comment by WasabiFan** [comment]
Another part of this task is to invite people to look at and contribute to the website repo. I think that people generally don't notice that this repository exists, so a means of getting people here who may want to help would be nice.
